debian/mariadb-plugin-connect.install

```diff
@@ -1,2 +1,3 @@
 etc/mysql/conf.d/connect.cnf  etc/mysql/mariadb.conf.d
 usr/lib/mysql/plugin/ha_connect.so
+usr/share/mysql/JdbcInterface.jar
```

Comment:
Why in `share` if cmake is installing in `plugin` for 10.3?
```
  SET (JAVA_SOURCES JdbcInterface.java)
  add_jar(JdbcInterface ${JAVA_SOURCES})
  INSTALL(FILES
    ${CMAKE_CURRENT_SOURCE_DIR}/JavaWrappers.jar
    ${CMAKE_CURRENT_BINARY_DIR}/JdbcInterface.jar
    DESTINATION ${INSTALL_PLUGINDIR} COMPONENT connect-engine)
ENDIF()
```

Reply:
@illuusio  I'm looking into Otto's `10.3-MDEV-26218` branch as an git remote, where I can easily cherry-pick/rebase his commits, so this is 10.3 tree it has snippet as above.